src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -21,334 +20,261 @@
-                        <Setter Target="GeneralView.(Grid.Row)" Value="1" />
-                        <Setter Target="GeneralView.Margin" Value="0" />
-                        <Setter Target="LinksPanel.(RelativePanel.RightOf)" Value="AboutImage"/>
-                        <Setter Target="LinksPanel.(RelativePanel.AlignTopWith)" Value="AboutImage"/>
-                        <Setter Target="AboutImage.Margin" Value="0,12,12,0"/>
-                        <Setter Target="AboutTitle.Visibility" Value="Collapsed" />
-                    </VisualState.Setters>
-                </VisualState>
-            </VisualStateGroup>
-        </VisualStateManager.VisualStateGroups>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="*"/>
-            <ColumnDefinition Width="Auto"/>
-        </Grid.ColumnDefinitions>
-        <Grid.RowDefinitions>
-            <RowDefinition Height="Auto"/>
-            <RowDefinition Height="Auto" />
-        </Grid.RowDefinitions>
-
-        <StackPanel Orientation="Vertical"
-                    x:Name="GeneralView"
-                    Margin="0,0,48,0"
-                    HorizontalAlignment="Left"
-                    MaxWidth="{StaticResource MaxContentWidth}">
-            <TextBlock x:Uid="Admin_Mode" 
+    <controls:SettingsPageControl x:Uid="About_PowerToys"
+                                  ModuleImageSource="ms-appx:///Assets/Modules/PT.png"
+                                  ModuleImageLink="https://aka.ms/powertoys">
+        <controls:SettingsPageControl.ModuleContent>
+            <StackPanel Orientation="Vertical">
+                <Button Click="OobeButton_Click"
+                        Style="{StaticResource AccentButtonStyle}"
+                        Margin="0,12,0,24">
+                    <Button.Content>
+                        <StackPanel Orientation="Horizontal">
+                            <FontIcon FontSize="13" Glyph="&#xF133;" />
+                            <TextBlock Margin="8,0,0,0"
+                               x:Uid="Oobe_Button"/>
+                    </StackPanel>
+                    </Button.Content>
+                </Button>
+                
+                
+                <TextBlock x:Uid="Admin_Mode" 
                        FontWeight="SemiBold"
                        TextWrapping="Wrap"
                        AutomationProperties.HeadingLevel="Level2"
                        Style="{StaticResource SubtitleTextBlockStyle}"/>
 
-            <TextBlock Text="{Binding Mode=TwoWay, Path=RunningAsText}"
+                <TextBlock Text="{Binding Mode=TwoWay, Path=RunningAsText}"
                        TextWrapping="Wrap"
                        Margin="{StaticResource SmallTopMargin}"/>
 
-            <Button x:Uid="GeneralPage_RestartAsAdmin_Button"
+                <Button x:Uid="GeneralPage_RestartAsAdmin_Button"
                     Margin="{StaticResource SmallTopMargin}"
-                    Style="{StaticResource AccentButtonStyle}"
```

Comment:
Is this change correct? Run as admin button has different style now - it's not blue, but dark grey

Reply:
Yeah, to make the OOBE button stand-out more.